gp-okta.conf

```diff
@@ -8,8 +8,9 @@ sms.okta = 0
 totp.okta = ABCDEFGHIJKLMNOP
 totp.google = ABCDEFGHIJKLMNOP
 gateway = Manual ny1-gw.example.com
+#cert = path-to-client-cert-as-unencrypted-pem-file.pem
 #openconnect_cmd = sudo openconnect
-openconnect_args =                    # optional arguments to openconnect
+openconnect_args =                    # optional arguments to openconnect, probably you might want to repeat the cert here (if used above) with --certificate=xxx
```

Comment:
Could this be added automatically by the script if`cert` is specified in the config file? If it's always required, we might as well only have one place it has to be configured...

Reply:
Good point. I also thought about that.
But probably one needs some other certificate for the openconnect call? Or only one for the "dance"? In the end I left it open, offering the most flexibility (only with the comment as hint) as it would be hard to "stop" such an automatism if the setup is different and only requires the certificate once. I think the `gp-okta.conf` is still small enough to mentally understand all of it.

Reply:
I confess I don't understand enough about your setup to understand why the two certificates are probably different. Why is that?

Reply:
Well, if you don't like it ... for me they are also identical -- if one needs that differently, he/she can raise a pull request lateron himself ;-)
-> Changed in fab69fe18ccc75c9e63dc1c94c96a47547aca4f4

Reply:
Sorry if it came off as such, but my last comment was not intended to goad you into doing what I'd initially requested.

I honestly don't understand how these certificates are used, and just wanted to make sure we're doing the right thing. I think that the flexibility you mentioned is a good reason to keep them separate config options as long as we expect someone to actually use it.

Are you happy with the current state of this PR, or do you still prefer them separate? I want this to be the correct decision, and not to be made based on what you think I want...

Reply:
I'm fine with the current state of this PR, simply as it works for me - in my use case there is only one certificate involved.

But: please see @lvml comment below. He made a valid point - not the application is relevant (python 'requests' vs. openconnect) - in fact the destination domain should be relevant.

As the s = requests.Session() from line 461 is used for vpn_url and okta_url altogether a separation of the certificates (if you want that) should probably be better based on the destination.